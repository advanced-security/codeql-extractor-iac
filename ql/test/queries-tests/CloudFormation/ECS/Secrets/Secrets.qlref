security/CloudFormation/ECS/Secrets.ql
