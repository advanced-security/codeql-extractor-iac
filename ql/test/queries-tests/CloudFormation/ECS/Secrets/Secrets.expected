| ecs.yml:27:13:31:6 | - Name: ... KEY_ID' | Containers must not pass secret thorugh environment variables |
